spec/support/vcr_cassettes/vaos/v2/systems/get_available_slots_vpg_200.yml

```diff
@@ -54,7 +54,7 @@ http_interactions:
         - max-age=63072000; includeSubDomains; preload
     body:
       encoding: UTF-8
```

Comment:
If this is a newly-recorded cassette, I would expect other items in the response to be different. Same with the other cassette

Reply:
Yes, this was manually edited it in order to add the new VPG fields that we've added.

I feel like the test itself provides value since we're able to verify that VPG is hit instead of VAOS, and that the structure of the data returned from VPG is correctly handled, but I can see how this isn't the same as an actually recorded cassette. Is there a preferred method to mocking the response in vets-api or is there an acceptable way to communicate that the response wasn't recorded?

Reply:
@devin-mccurdy manually editing cassettes has bitten us before. We thought the response was in one shape and the tests were passing, so we were happy, but it turned out the code (non-test) changes we'd made caused an outage. 

So it's always best to re-record if possible, especially if the code changes are risky. What's the level of risk here?

Reply:
Thanks, that makes sense.

I'd say relatively low risk, we've designed VPG to be a drop in replacement of VAOS, so the additional fields will be the only difference between responses, and we'll be testing this thoroughly and have good control w/ the feature flags.